upi/vsphere/lb/haproxy.tmpl

```diff
@@ -32,6 +32,12 @@ backend api-server
     %{ for addr in api ~}
```

Comment:
don't we now have multiple endpoints for backend,
one based on TCP check, while the new ones based on HTTPS check

Reply:
I was thinking testing it both in the network layer and the http layer might help figuring out where exactly the installation went wrong. Let me know if it's not required.